css/main.css

```diff
@@ -195,4 +195,25 @@ textarea {
     h3 {
         page-break-after: avoid;
     }
+}
+
+a{
+	text-decoration: none;
+}
+
+li{
+  display: inline;
+  margin-left: 170px;
+  font-size: 30px;
+}
+
+#left{
+	margin-left: 700px;
+}
+
+.menu{
+	width: 100%;
+	height: 75px; 
+	background-color: #FFEFD5;
+    padding-top: 8px;
 }
```

Comment:
You want to strive for consistency in the file. Make sure the same amount of spacing is used